app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,10 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
+    <!-- Needed if app targets Android 5.0 (API level 21) or higher -->
+    <uses-feature android:name="android.hardware.location.gps" />
```

Comment:
Do we need this? I thought the ones above already took care of the needed permissions?